Providers/Profile.swift

```diff
@@ -99,7 +88,6 @@ protocol Profile: AnyObject {
     var searchEngines: SearchEngines { get }
     var files: FileAccessor { get }
     var history: HistoryFetcher { get }
-    var metadata: Metadata { get }
```

Comment:
Isn't this used here 🤔 

<img width="611" alt="Screen Shot 2022-09-02 at 4 00 56 PM" src="https://user-images.githubusercontent.com/8919439/188227719-cbea762a-6e4e-41d2-8a7b-ad008e3bf8d0.png">

Reply:
Or you just don't want it to be part of the protocol?

Reply:
If it's not referenced by any via the protocol then it shouldn't be in the protocol imo.